src/page-view.ts

```diff
@@ -24,15 +25,21 @@ interface ScrollProperties {
     $prev_pageview_max_content_percentage?: number
 }
 
+interface PageViewEventProperties extends ScrollProperties {
+    $prev_pageview_pathname?: string
```

Comment:
We might want to update `isURLNormalizeable()` in the main repo to include this property